frontend/tsconfig.config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@tsconfig/node18/tsconfig.json",
+  "extends": "@tsconfig/node20/tsconfig.json",
```

Comment:
This should be node22. Would be surprised if this didn't explode at the linting/tsc stage without node20 present in the explicit dependencies. This might be sliding by because of one of our other dependencies down the chain using node20.

Reply:
Updated to `@tsconfig/node22` for both FE and BE.